content/millennium/r4/proprietary-codes.md

```diff
@@ -190,6 +203,28 @@ This code set is client definable for different patient types such an Outpatient
       "userSelected": true
     }
 
+#### Code Set 72 Clinical Event Codes
+
+This code set is extremely large and highly customized by clients. It maintains all the various clinical events such as vital signs, laboratory results, documents, and immunizations.
+
+    { 
+      "system": "https://fhir.cerner.com/2c400054-42d8-4e74-87b7-80b5bd5fde9f/codeSet/72", 
```

Comment:
Changed the tenant key in the URL's everywhere applicable to <EHR Source id>
https://github.com/cerner/fhir.cerner.com/pull/255/commits/3fa313c0f09253444d22d4ea494e61e108042e75#diff-f1a7812a3d6e9f4049fb6005eecfb1f5L211

Reply:
+1